### PR TITLE
Enhancement: Fix CSS again

### PR DESCRIPTION
**Pull request by EmeraldSnorlax**
https://user-images.githubusercontent.com/43452521/107768758-de463b80-6d2e-11eb-85d9-b2222601e8da.mp4

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -155,9 +147,30 @@ func getHtml() string {
...
+	newLastBeat := time.Now().Unix()
+	currentBeatDifference := newLastBeat - lastBeat
+
+	if currentBeatDifference > missingBeat {
+		missingBeat = currentBeatDifference
+	}
 
-	return htmlWithAbsence
+	lastBeatStr := strconv.FormatInt(newLastBeat, 10)
+	missingBeatStr := strconv.FormatInt(missingBeat, 10mmm  )
```
Comment (EmeraldSnorlax) [inline main.go:168]: is this intentional?

Comment (l1ving) [reply main.go:168]: Your merge meme did it

Comment (EmeraldSnorlax) [reply main.go]: oof

File: www/index.html [hunk 2]
```diff
@@ -16,9 +16,10 @@
         <div class="spacer"></div>
         <div class="pure-g preamble">
             <div class="pure-g-u-0 pure-u-lg-1-3"></div>
-            <div class="pure-u-1 pure-u-lg-1-3">
+            <div class="pure-u-1 pure-u-lg-1-2">
```
Comment (EmeraldSnorlax) [inline www/index.html:19]: if you want this wider, maybe make the grid 1/5 - 3/5 - 1/5

**Comment by l1ving** [comment]
Thanks!

**Pull request merged**